admin/spec/controllers/pupil-pin.spec.js

```diff
@@ -5,15 +5,16 @@ const httpMocks = require('node-mocks-http')
 const sinon = require('sinon')
 require('sinon-mongoose')
 
+const checkStartService = require('../../services/check-start.service')
 const config = require('../../config')
+const dateService = require('../../services/date.service')
+const pinGenerationService = require('../../services/pin-generation.service')
+const pinService = require('../../services/pin.service')
+const pupilDataService = require('../../services/data-access/pupil.data.service')
+const qrService = require('../../services/qr.service')
 const School = require('../../models/school')
```

Comment:
how come we still reference this model and need sinon-mongoose?

Reply:
Good spot.  Looks like a mistake.  I've removed the mongoose model now.